### PR TITLE
Wrd html text box editor

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

**Review by DimitarNikolovv06 — DISMISSED** [review]
Seems good!

File: designing-reports/report-designer-tools/web-report-designer/tools/wrd-htmltextbox-editor.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The editor lets you easily add [Embedded Expressions in HtmlTextBox]({%slug htmltextbox_embedded_expressions%}) through the list of predefined and dynamic elements that can be used in your expression. A list of expression element categories is located in the lower left-hand list of the editor, the elements for each category are located in the lower middle list, and descriptions are displayed in the lower right-hand panel.
+
+![Look at the HtmlTextBox editor of the Web Report Designer.](images/wrd-htmltextbox-editor-screenshot.png)
+
+To add an embedded expression to the HtmlTextBox content double-click the list of elements in the bottom middle list. This will add the corresponding dynamic object wrapped in curly brackets. You may further edit the content inside the brackets manually by adding other dynamic objects and relations between them. The entire embedded expression will be evaluated and replaced in runtime by the Reporting engine.
```
Comment (mzanev) [inline designing-reports/report-designer-tools/web-report-designer/tools/wrd-htmltextbox-editor.md:19]: Isn't "double-click an element from the bottom middle list" more accurate than "double-click the list of elements in the bottom middle list" here?

**Review by mzanev — APPROVED** [review]
Left a minor nitpick but the article looks good regardless.
Thank you for writing it!

**Review by mzanev — APPROVED** [review]
Thank you for implementing the changes!

**Pull request merged**